content/docs/guides/overview.md

```diff
@@ -3,22 +3,17 @@ title: Introduction
 subtitle: New to Cucumber? Start here!
 weight: 1000
 ---
+Cucumber is a tool that supports [Behaviour-Driven Development(BDD)](/docs/bdd).
+If you're new to Behaviour-Driven Development read our [BDD introduction](/docs/bdd)
+first.
 
-{{% note "Before you get started" %}}
-If you're new to Behaviour-Driven Development (BDD) (or if you think it's a testing technique 😏) read
-our [BDD introduction](/docs/bdd) first.
-{{% /note %}}
-
+# What is Cucumber?
 Ok, now that you know that BDD is about discovery, collaboration and examples
 (and not testing), let's take a look at Cucumber.
 
-# What is Cucumber?
-
-Cucumber is a tool that supports [Behaviour-Driven Development(BDD)](/docs/bdd).
-
-Cucumber reads executable specifications written in plain text and validates that the
-software does what those specifications say. The specifications consists of multiple
-*examples*, or *scenarios*. For example:
+Cucumber reads executable specifications written in plain text and validates
+that the software does what those specifications say. The specifications
+consists of multiple *examples*, or *scenarios*. For example:
 
 ```gherkin
```

Comment:
Tbh I have no idea what this example means. Could we replace it with something people would (more) easily recognise?